apps/studio/state/notifications.ts

```diff
@@ -0,0 +1,69 @@
+import { proxy, snapshot, useSnapshot } from 'valtio'
+
+export const notificationsState = proxy({
+  filterStatuses: [] as string[],
+  filterPriorities: [] as string[],
+  filterOrganizations: [] as string[],
+  filterProjects: [] as string[],
+  get numFiltersApplied() {
+    return [
+      ...notificationsState.filterStatuses,
+      ...notificationsState.filterPriorities,
+      ...notificationsState.filterOrganizations,
+      ...notificationsState.filterProjects,
```

Comment:
I have a feeling this might have to use `this` here to be reactive

```suggestion
      ...this.filterStatuses,
      ...this.filterPriorities,
      ...this.filterOrganizations,
      ...this.filterProjects,
```